test/e2e/config/e2e_conf.yaml

```diff
@@ -1,10 +1,6 @@
 managementClusterName: caprke2-e2e
 
 images:
-  - name: ghcr.io/rancher/cluster-api-provider-rke2-bootstrap:v0.5.0
-    loadBehavior: tryLoad
-  - name: ghcr.io/rancher/cluster-api-provider-rke2-controlplane:v0.5.0
-    loadBehavior: tryLoad
```

Comment:
Even though it is not important to have this lines, they allow to pre-load images to the cluster, instead of pulling them every time. This speeds up local e2e tests execution.

Reply:
I removed them solely because I wanted to be aligned with upstream https://github.com/kubernetes-sigs/cluster-api/blob/main/test/e2e/config/docker.yaml#L14-L25 and also not getting confused seeing the duplicated images one with actual released version and second with `dev` tag.